perses/rjmc/geometry.py

```diff
@@ -922,13 +877,96 @@ def _torsion_logp(self, growth_context, torsion, positions, r, theta, phi, beta,
         torsion_logp = logp_torsions[phi_idx] - np.log(2*np.pi / n_divisions) # convert from probability mass function to probability density function so that sum(dphi*p) = 1, with dphi = (2*pi)/n_divisions.
         return torsion_logp
 
+class SplineTorsionGeometryEngine(FFAllAngleGeometryEngine):
+    """
+    A class using splines for the torsion part.
+    """
+
+    def __init__(self):
+        raise NotImplementedError("This functionality is incomplete.")
+
+    def _torsion_logp(self, growth_context, torsion, positions, r, theta, phi, beta, n_divisions=360):
+        """
+        Use a spline fit to the torsion profile to get a logp
+        Parameters
+        ----------
+        growth_context
```

Comment:
These function docstrings are incomplete. Parameters should be documented in [numpy format](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#docstring-standard). I think this is important even for private functions, since there are multiple developers.